### PR TITLE
Overhaul of timeslider verbiage

### LINKED ISSUE #722: Map - timeslider tooltip verbiage should mention that it is visualising data for the whole world not just the viewport
Probably just by adding `(anywhere in the world)` to the first bullet item?

### PR DESCRIPTION
**Pull request by bobular**
Fixes #722 
Screenshot from 2024-02-16 17-01-30

### CODE REVIEW

**Comment by bobular** [comment]
Just saw it says "has data that... has data for".  Will amend!

File: packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx [hunk 1]
```diff
@@ -196,12 +196,25 @@ export default function TimeSliderQuickFilter({
...
-          <li>Permanent filters are applied if applicable</li>
+          <li>is located anywhere on Earth*, not just currently in view</li>
           <li>
-            You currently have {filters?.length} active permanent filter(s)
+            has a value for <b>{variableMetadata?.variable.displayName}</b>
           </li>
+          <li>satisfies any filters you have applied</li>
+          <li>
+            has values for the variable currently displayed on markers,
+            including custom-configurations if applicable
```
Comment (dmfalke) [inline packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx:208]: I think we can lose "if applicable"

File: packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx [hunk 2]
```diff
@@ -196,12 +196,25 @@ export default function TimeSliderQuickFilter({
...
+          </li>
+          <li style={{ listStyleType: 'none' }}>
```
Comment (dmfalke) [inline packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx:210]: Ideally this would not be a list item, since it isn't semantically speaking.

**Review by dmfalke — APPROVED** [review]
Left some suggestions, but fine as-is

File: packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx (hunk 2 above)
Comment (bobular) [reply packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx:210]: For screen-reader purposes?

Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx:210]: Yep, that's the main reason

Comment (bobular) [reply packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx:210]: I wanted it to be the same indentation level but not have to eyeball the indentation level. But putting it back to the <p> level looks fine, especially with parentheses.

Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx]: I was about to say that I also don't like that it's indented with the rest of the list. To me, it gets lost. I expect "footnotes" to appear not indented.

**Pull request merged**